reference/api/facet_search.mdx

```diff
@@ -0,0 +1,59 @@
+# Facet search
+
+The `/facet-search` route allows you to search for facet values. Facet search supports [prefix search](/learn/advanced/prefix) and [typo tolerance](/learn/configuration/typo_tolerance). The returned hits are sorted lexicographically in ascending order. You can configure how facets are sorted using the [`sortFacetValuesBy`](/reference/api/settings#faceting-object) property of the `faceting` index settings.
+
+<Capsule intent="Facet search on numbers">
+Internally, Meilisearch represents numbers as [`float64`](https://en.wikipedia.org/wiki/Double-precision_floating-point_format). Since they lack precision and can be written in different ways, Meilisearch does not support facet search on numbers.
+</Capsule>
+
+## Perform a facet search
+
+Search for a facet value within a given facet.
+
+<RouteHighlighter method="POST" route="/indexes/{index_uid}/facet-search"/>
+
+<Capsule intent="warning">
+This endpoint will not work without first explicitly adding attributes to the [`filterableAttributes`](/reference/api/settingss#update-filterable-attributes) list. [Learn more about facets in our dedicated guide.](/learn/fine_tuning_results/faceted_search#configuring-and-using-facets)
+</Capsule>
+
+<Capsule intent="warning">
+Multi-word facets are not supported as Meilisearch only considers one term for `facetQuery`. For example, `Jane Austen` is considered a single term, searching for `Jane` will return `Jane Austen`, but searching for `Austen` will not return `Jane Austen`.
+</Capsule>
+
+### Body
+
+| Name                                                              | Type                                                               | Default value | Description                                                                                                                                                                                                                                                                                                     |
+| :---------------------------------------------------------------- | :----------------------------------------------------------------- | :------------ | :-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| **`facetName`** *                                                 | String                                                             | `null`        | Facet name to search values on                                                                                                                                                                                                                                                                                  |
```

Comment:
The `attributesToSerachOn` and `vector` (but it's experimental) are also accepted fields.
https://www.notion.so/meilisearch/Search-for-facet-values-PRD-c9499f0654dd4f06ad786730222f4ad2?pvs=4#7914a3c2ace647bfabb88a45119b2ffc

Reply:
I added `attributesToSearchOn`, but I'm not sure about `vector`.

What do you think @guimachiavelli?

Reply:
I'm pretty sure we don't want to add `vector` :)

Reply:
`vector` is an experimental feature. We should keep documentation of experimental features enclosed in their own sections, as we otherwise risk huge headaches whenever anything changes or gets cancelled. It can also muddy the waters, as it might become harder for users to identify what's stable and ready for production usage.